### ISSUE TITLE
Font hard to read

### ISSUE THREAD

**Issue by darobin** [w3c/developer-tools#16]
The font size is very small, it is hard to read even for 20/20 people, and has clear accessibility issues. The contrast is also insufficient, notably of the part on the greyish background, as well as of links.

**Comment by gbaudusseau** [comment]
font size updated.

**Comment by tripu** [comment]
Looking better now, @guibbs.
